packages/syft/src/syft/core/adp/vectorized_publish.py

```diff
@@ -155,7 +157,20 @@ def publish(
         # Step 2: Calculate the epsilon spend for this query
 
         # rdp_constant = all terms in Theorem. 2.7 or 2.8 of https://arxiv.org/abs/2008.11193 EXCEPT alpha
-        rdp_constants = compute_rdp_constant(rdp_params, private=private)
+        if any(np.isnan(l2_norms)):
+            if any(np.isnan(l2_norm_bounds)) or any(np.isinf(l2_norm_bounds)):
```

Comment:
MAYBE: We could also raise the error when we extract min/max values if the min/max values hold nan values.

Reply:
Unrelated to Rasswanth's comment- but would the use of `any()` fail here if `l2_norms` is a singleton, like the result of `sum()`?